.github/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 <h3>New features since last release</h3>
 
+* The `qml.math` module now supports JAX.
```

Comment:
Let's move this down to 'improvements'? I don't want to advertise the `qml.math` module too much until (a) it is more battle-tested, and (b) it is better optimized.

Reply:
```suggestion
* The `qml.math` module now supports JAX.
  [(#985)](https://github.com/XanaduAI/software-docs/pull/274)
```